utils/ccformat.py

```diff
@@ -43,6 +43,8 @@ def runTidy(args):
                            "LLVM_BUILD environment variable."
       return 1
 
+    coreclrbuild = expandPath(args.coreclr_build)
```

Comment:
we need a check like above to confirm that coreclr_build is not none. Also, is this the path to just the coreclr directory or some other path? I need this to make the job work.